From 93c1d5fe7bb7a62ef884eb41b505b2809d1704b6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20P=2E=20Berrang=C3=A9?= <berrange@redhat.com>
Date: Thu, 8 Aug 2019 13:42:24 +0100
Subject: [PATCH] network: fix ability to use openvswitch with vlans
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Use the correct enum constant when validating vlan usage.
This fixes a merge error in

  commit 6cb0ec48bd95c95489a987e05a88e8bcf1f9109c
  Author: Daniel P. Berrangé <berrange@redhat.com>
  Date:   Mon Sep 3 17:34:22 2018 +0100

    network: convert networkAllocateActualDevice to virNetworkPortDef

Reviewed-by: Michal Privoznik <mprivozn@redhat.com>
Signed-off-by: Daniel P. Berrangé <berrange@redhat.com>
---
 src/network/bridge_driver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/network/bridge_driver.c b/src/network/bridge_driver.c
index 41fa89a..72e1186 100644
--- a/src/network/bridge_driver.c
+++ b/src/network/bridge_driver.c
@@ -4814,7 +4814,7 @@ networkAllocatePort(virNetworkObjPtr obj,
         if (!(port->plugtype == VIR_NETWORK_PORT_PLUG_TYPE_HOSTDEV_PCI ||
               (port->plugtype == VIR_NETWORK_PORT_PLUG_TYPE_DIRECT &&
                port->plug.direct.mode == VIR_NETDEV_MACVLAN_MODE_PASSTHRU) ||
-              (port->plugtype == VIR_DOMAIN_NET_TYPE_BRIDGE &&
+              (port->plugtype == VIR_NETWORK_PORT_PLUG_TYPE_BRIDGE &&
                port->virtPortProfile &&
                port->virtPortProfile->virtPortType == VIR_NETDEV_VPORT_PROFILE_OPENVSWITCH))) {
             virReportError(VIR_ERR_CONFIG_UNSUPPORTED,
-- 
2.22.0

